### DISCUSSION TITLE
一个像star-history那样可以使用户将图表自由嵌入README等文档和网页中的svg服务

### DISCUSSION THREAD

**Discussion by tyn1998** [X-lab2017/open-wonderland#238] [2️⃣ 聊项目]
Original post: https://github.com/hypertrons/hypertrons-crx/issues/532

郭旭东老师之前在Hypercrx中提过这个需求，我简单想了想，认为要支持像star-history这样的服务，应该需要另外创建一个项目和一套技术栈做这件事情，因此关闭了Issue 532，并转述到X-lab Discussion中，在这里讨论。

**Comment by tyn1998** [comment]
我看到 #235 的(3)社会服务与企业合作里也提到了“OpenDigger 开放数据的 SVG 图片生成工具”。

**Reply by sunny0826** [reply]
同意独立创建一个以 OpenDigger 为源数据的新项目来生成静态 SVG 图片，这样就可以直接在其余项目的 README 中直接引用 chart

**Reply by TieWay59** [reply]
https://twitter.com/OSSInsight/status/1697637054958706986?s=19


oss insight 八月已经做了两版。🌝

**Reply by TieWay59** [reply]
他们还都是散件SVG，没有搞大综合的仪表盘。我估计也快了。📰👀

**Reply by TieWay59** [reply]
![image](https://github.com/X-lab2017/open-wonderland/assets/40034603/e90a7fbe-5634-4e47-827e-34fe745922be)

GitHub Social Circle 这个图看上去很有意思，因为实际上我们的数据可以做的更好。而且这个图的绘制网站已经停用了。

**Reply by tyn1998** [reply]
这个形式真不错！创建了一个Feature Issue：https://github.com/hypertrons/hypertrons-crx/issues/735

**Comment by tyn1998** [comment]
开个头：https://github.com/X-lab2017/open-widgets

可以先学习一下star-history的源码：https://github.com/star-history/star-history
